### PR TITLE
change the README.md to use examples with oc/podman

### PR DESCRIPTION
**Pull request by pkubatrh**

### CODE REVIEW

File: 8.0/root/usr/share/container-scripts/mysql/README.md [hunk 1]
```diff
@@ -187,16 +188,26 @@ location is `/etc/my.cnf` but you can change it to `/etc/mysql/my.cnf` by settin
...
+
+```
+$ oc new-app mysql:8.0~./image-configuration/ --name my-mysql-rhel7
+```
+
+or via s2i:
 
 ```
-$ s2i build ~/image-configuration/ rhscl/mysql-80-rhel7 my-mysql-rhel7
+$ s2i build ./image-configuration/ rhscl/mysql-80-rhel7 new-postgresql
```
Comment (kubco2) [inline 8.0/root/usr/share/container-scripts/mysql/README.md]: Probably mistake

Comment (kubco2) [reply 8.0/root/usr/share/container-scripts/mysql/README.md]: it is in mariadb container too

File: 5.7/root/usr/share/container-scripts/mysql/README.md [hunk 2]
```diff
@@ -187,16 +188,25 @@ location is `/etc/my.cnf` but you can change it to `/etc/mysql/my.cnf` by settin
...
 ```
-$ s2i build ~/image-configuration/ rhscl/mysql-57-rhel7 my-mysql-rhel7
+$ oc new-app mysql:5.7~./image-configuration/ --name my-mysql-rhel7
 ```
 
-The directory passed to `s2i build` can contain these directories:
+or via s2i:
+
+```
+$ s2i build ./image-configuration/ rhscl/mysql-57-rhel7 new-postgresql
```
Comment (kubco2) [inline 5.7/root/usr/share/container-scripts/mysql/README.md]: also

File: 8.0/root/usr/share/container-scripts/mysql/README.md (hunk 1 above)
Comment (pkubatrh) [reply 8.0/root/usr/share/container-scripts/mysql/README.md]: Definitely a mistake, will fix in both. Thanks for the review!

File: 5.7/root/usr/share/container-scripts/mysql/README.md [hunk 3]
```diff
@@ -187,16 +188,25 @@ location is `/etc/my.cnf` but you can change it to `/etc/mysql/my.cnf` by settin
...
+available via `mysql:5.7` imagestream tag in Openshift.
 
-For example, to build a customized MariaDB database image `my-mysql-rhel7`
-with a configuration in `~/image-configuration/` run:
+For example, to build a customized MySQL database image `my-mysql-rhel7`
+with a configuration in `image-configuration/` run:
 
 ```
-$ s2i build ~/image-configuration/ rhscl/mysql-57-rhel7 my-mysql-rhel7
+$ oc new-app mysql:5.7~./image-configuration/ --name my-mysql-rhel7
```
Comment (hhorak) [inline 5.7/root/usr/share/container-scripts/mysql/README.md]: I don't think local dirs work in OpenShift, what about to use:
```
oc new-app mysql:5.7~https://github.com/yourorg/my-own-mysql-configuration.git --name my-mysql-rhel7
```
or even
```
oc new-app mysql:5.7~https://github.com/sclorg/mysql-container.git --context-dir=examples/extend-image --name my-mysql-rhel7
```

Comment (pkubatrh) [reply 5.7/root/usr/share/container-scripts/mysql/README.md]: Ah right, missed that it is not explicitly stated that the `image-configuration` dir must be an accessible git repository containing the configuration. Thanks for spotting this!

I will change this to the longer example.

**Comment by pkubatrh** [comment]
@hhorak @kubco2 PTAL

**Comment by kubco2** [comment]
LGTM

**Comment by pkubatrh** [comment]
Thanks for the review! Merging

**Pull request merged**